playbooks/notifications/email-notify-single-user.yml

```diff
@@ -1,7 +1,5 @@
 ---
 
-- include_vars:
```

Comment:
@bvkin Why are we getting rid of the `email_content_file` ?

Reply:
Playbooks/roles don't seem to use this variable as far as I can tell. Maybe there's something I missed here though?

Also, for using dynamic inventories like with omp it won't really make sense to use this because we're not storing email content locally.

Reply:
I too am concerned with this change. It will certainly break current use of this playbook (we use it when onboarding users to environments, and I'm about to run this for next weeks session). Hence, I'd suggest not "breaking" this as-is, but rather look to include a new playbook with the desire functionality if this doesn't work as-is for your specific needs. 

It  may be better to jump on a call to discuss details, however.

Reply:
@bvkin just as a FYI - the expectation around the `email_content_file` is that the inventory used with the playbook sets it.

Reply:
OK, probably best to just close this pr then. I will make a new playbook to work with the dynamic inventory.